### DISCUSSION TITLE
MinIO vlans

### DISCUSSION THREAD

**Discussion by app13man** [nhost/hasura-storage#149] [General]
Hello,

New to the group, and new to MinIO. Have been tasked to provision object storage in our enviornment. Going thru the documentation, can't seem to find anything mentioning if MinIO can support multiple vlans? Anybody here have experience with that or have an answer? Thank you inadvance.

**Comment by dbarrosop** [comment]
Is this question about `hasura-storage` or `minio`? Just clarifying because this is `hasura-storage`'s repo, not minio :)

In any case, services don't need to support vlans, what you need to do is be able to bind a service to multiple IP addresses (one belonging to each vlan). Worst case, if you bind a service to `0.0.0.0` or `::` it will bind to all interfaces in the host.
